packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -238,16 +239,11 @@ export const Subscriptions = ({
         />
       )}
 
-      {updatePaymentStatus.result && showPaymentSuccessAlert && (
+      {updatePaymentStatus.result && (
         <AlertBar className="alert alertSuccess alertCenter">
           <span data-testid="success-billing-update" className="checked">
             Your billing information has been updated successfully
           </span>
-          <span
-            data-testid="clear-success-alert"
-            className="close"
-            onClick={clearSuccessAlert}
-          />
```

Comment:
This and its associated state, onclick handler, and styles are removed because the alert bar now self-dismisses.

Related to https://github.com/mozilla/fxa/issues/3187.